src/vs/workbench/contrib/terminal/browser/media/xterm.css

```diff
@@ -208,7 +209,3 @@
 	z-index: 2;
 	position: relative;
 }
-
-.xterm.terminal.hide {
-	visibility: hidden;
-}
```

Comment:
@meganrogge FYI xterm.css is upstream styles only, terminal.css is for overrides and workbench